src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div 
+      class="card" 
+      data-qa="card"
+    >
+      <img 
+        class="card__img" 
+        src="./images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27"
+      >
+
+      <h1 class="card__title ">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+      <span class="card__product-code">Product code: 195434</span>
+
+      <div class="card__rating">
+        <div class="card__stars card__stars--4">
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+
+          <div class="card__star"></div>
+        </div>
```

Comment:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>

          <div class="stars__star"></div>

          <div class="stars__star"></div>

          <div class="stars__star"></div>

          <div class="stars__star"></div>
        </div>
```
цей блок треба перевикористати із таски про зірочки, згідно з завданням це має бути окремий БЕМ блок